zones.h

```diff
@@ -9,8 +9,13 @@
 extern "C" {
 #endif
 
+typedef struct {
+  const char *name;
+  const char *posix_str;
+} micro_tz_db_pair;
 
 const char * micro_tz_db_get_posix_str(const char * name);
+const micro_tz_db_pair *get_tz_db(void);
```

Comment:
can you show your use case of expanding the API? the struct typedef and static array of structs are meant to be implementation only, i don't see any value in exposing them